.cirrus.yml

```diff
@@ -558,6 +558,44 @@ unit_test_task:
     always: *logs_artifacts
 
 
+unit_test_windows_task:
+    name: &std_name_fmt "$TEST_FLAVOR $PODBIN_NAME $DISTRO_NV $PRIV_NAME $TEST_ENVIRON ${CI_DESIRED_DATABASE}"
```

Comment:
The name is rather confusing as most of the options have nothing to do with unit tests it should follow the existing unit test name.
I suggest you name this "Unit tests on Windows"